### ISSUE TITLE
How to pass reporter options

### ISSUE THREAD

**Issue by jvanoostveen** [SitePen/remap-istanbul#31]
We use some custom reporter options for the threshold colors, and I was wondering how I pass these to the remap-istanbul to the reports. As after the remap, the coverage color thresholds are back to the defaults.

Now the 'normal' istanbul is called (`instanbul.writeReports(config.istanbul.reports)`) with the following as the value of the config.istanbul.reports:
```javascript
{
  dir: 'build/report',
  reporters: [
    'lcov',
    'json'
  ],
  reportOpts: {
    watermarks: {
      statements: [75, 100],
      lines: [75, 100],
      functions: [75, 100],
      branches: [75, 100]
    }
  }
}
```

Is it possible to pass the options for the reporters when remapping the output? And if so, how?

BTW: I'm using the gulp remap-istanbul.

**Comment by kitsonk** [comment]
Currently you can't, as the write report creates [it's own object literal](https://github.com/SitePen/remap-istanbul/blob/30b67ad3f24ba7e0da8b8888d5a7c3c8480d48b2/lib/writeReport.js#L38).  It should be fairly easy to add the functionality though and it makes sense.

**Comment by kitsonk** [comment]
Closed via d07ac8c439a724d0e3aeb748938ee73a07451614 thanks @jvanoostveen
